interface/src/index.html

```diff
@@ -15,12 +15,29 @@
     <!-- font awesome from BootstrapCDN -->
     <link href="http://netdna.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.css" rel="stylesheet">
 
+
+    <!-- Required CSS for ui-slider -->
+    <link rel="stylesheet"
+          href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/smoothness/jquery-ui.css">
+
     <!-- compiled CSS --><% styles.forEach( function ( file ) { %>
     <link rel="stylesheet" type="text/css" href="/static/<%= file %>" /><% }); %>
 
+    <!-- Required js for ui-slider
+         FIXME: Why adding these two libraries in build.config.js does not work?
```

Comment:
Yes, they are for the ui-slider widget. I tried both minified and non-minified version in **build.config.js**, but the slider never showed up.